.github/workflows/npm-publish.yml

```
@@ -1,33 +1,30 @@
-# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
-# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
-
-name: Node.js Package
-
-on:
-  release:
-    types: [created]
+---
+name: Publish to npm
+on: { release: { types: [created] } }
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)